SMLHelper/BiomeThings/FloatingOrigin.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Based on the Unity Wiki FloatingOrigin script by Peter Stirling
+// URL: http://wiki.unity3d.com/index.php/Floating_Origin
+
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+internal class FloatingOrigin : MonoBehaviour
```

Comment:
This looks very expensive. Please make a toggle option for `FloatingOrigin` and default it to `false`.

Reply:
Alright.

Reply:
I tested this and it's well noticeable even at 3000,0,3000.

Reply:
It's about as noticeable as when you're at 2000 axis of any direction. Which is only noticeable when you're paying real attention to what's flickering (beginning of (noticeable) precision loss is at around 2000, then at 5000 everything starts going crazy and unplayable). 
Again, this is expensive and not any pc can handle it smoothly so a toggle would be great.

Reply:
One problem with that, for some reason without origin shifting, entities stop spawning, and I'm not sure why this is.

Reply:
I'll add the toggle nonetheless though.

Reply:
Oh apparently I already added it and I forgot.